### ISSUE TITLE
brya: Scrolling up/down by Trackpoint stops working at some point on some web pages

### ISSUE THREAD

**Issue by 130s** [kinu-garage/hut_10sqft#894]
# Issue
Around the time where I installed `Recent Tabs` extension https://github.com/kinu-garage/hut_10sqft/issues/890#issuecomment-1712781875, I started \
seeing on some (not all) web pages on the same Chrome browser that scrolling up/down initially works but at some point it gets disabled. So far I've seeing in the following pages.
- gmail.com
- superuser.com

# Workaround
The following has been functioning:
- o Restart Chrome browser.
- x Disabling the k/b shortcut for `Recent Tabs` seems unnecessary (this indicates that the extension sets `ctrl-q` by default to activate the tab switcher popup window).

**Comment by 130s** [comment]
I just experienced it again twice on 2 different webpages.
Case-1: Github.com
- Trigger: Unknown. I was editing a comment on a GitHub ticket https://github.com/kinu-garage/hut_10sqft/issues/890#issuecomment-1712781875
- Workaround:
   - x Reload the page.
   - o Restart the program.

Case-2: Gmail.com
- Trigger: Unknown. No idea.
- Workaround:
   - x Reload the page.
   - o Restart the program.

**Comment by 130s** [comment]
Just saw it on Amazon.com.

**Comment by 130s** [comment]
This happens unbearably too frequently. To isolate the issue I disabled `Recent Tabs` for now.

**Comment by 130s** [comment]
> This happens unbearably too frequently. To isolate the issue I disabled `Recent Tabs` for now. Restarted Chrome browser.

On Amazon.com it still occurs. So `Recent Tabs`  has an allibi? Re-enabling it.

**Comment by 130s** [comment]
I'm leaning toward reporting this as a bug of `Recent Tabs`, as a result of a small isolation where I disabled RT and I stopped seeing that GMail stops taking scroll.

Then, on the RT's forum I was told my Chrome is dev version, not released version. https://jasonsavard.com/Chrome-Channel-Releases

- Stable 116.0.5845.210
- Beta 117.0.5938.55
- Dev(You) 118.0.5993.10

I have little motivation to use a bleeding edge version of Chrome.
Will separately handle the browser version.
